README.md

```diff
@@ -1,192 +1,165 @@
-[![Total alerts](https://img.shields.io/lgtm/alerts/g/MeasureAuthoringTool/MeasureAuthoringTool.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/MeasureAuthoringTool/MeasureAuthoringTool/alerts/) [![Language grade: Java](https://img.shields.io/lgtm/grade/java/g/MeasureAuthoringTool/MeasureAuthoringTool.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/MeasureAuthoringTool/MeasureAuthoringTool/context:java)
-
-# Measure Authoring Tool
+# Measure Authoring Tool (MAT)
 
 ## Installation
-The Measure Authoring Tool (MAT) was designed using many open source products including the Google Web Toolkit (GWT) framework, Java JDK, MySQL and Eclipse. GWT allows a developer to write client side code in Java and GWT converts it to JavaScript. The MAT uses MySQL as its backend database server and the IDE is Eclipse for Java EE Developers.
+The Measure Authoring Tool (MAT) was designed using many open source products including, the Google Web Toolkit (GWT) 
```

Comment:
I think that Andrew wanted to merge README and README_SB files.. 
The one with SB suffix has priority over regular README

Reply:
This one is actually the merged one. README_SB.md was deleted.

Reply:
oh, great